### PR TITLE
Fix typos in README

### PR DESCRIPTION
**Pull request by alco**
Hi,

These are just a bunch of fixes for typos I saw while reading the Readme. There are also two relative links in the Readme that don't lead anywhere, I wasn't sure how to fix them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,30 +20,32 @@ Two specific shortcomings that the ICU message format addresses:
...
+Mariée
 ```
-In `Gettext` this requires individual messages and conditional code in the application in order to present the correct message to an audience.  This is compounded by the fact that some languages have more than two g[rammatical genders](https://en.wikipedia.org/wiki/Grammatical_gender) (most have been two and four but but some are attested with up to 20.
 
-The ICU message format provides a mechanism (the [choice format](#Choice_format)) that helps translator and UX designers implement a single message to easily encapsulate messages conditional on grammatical gender (or any other selector)
+In `Gettext` this requires individual messages and conditional code in the application in order to present the correct message to an audience.  This is compounded by the fact that some languages have more than two [grammatical genders](https://en.wikipedia.org/wiki/Grammatical_gender) (most have two and four but some are attested with up to 20).
+
+The ICU message format provides a mechanism (the [choice format](#Choice_format)) that helps translator and UX designers implement a single message to easily encapsulate messages conditional on grammatical gender (or any other selector).
```
Comment (alco) [inline README.md:36]: The `[choice format](#Choice_format)` link does not lead anywhere.

File: README.md [hunk 2]
```diff
@@ -20,30 +20,32 @@ Two specific shortcomings that the ICU message format addresses:
...
 
 * English has two plural forms: singular and plural
-* French applies the singular rule to two values and a plural form to larger groupings
+* French applies the singular rule to two values (0 and 1) and a plural form to larger groupings
 * Japanese does not differentiate
 * Russian has 4 categories
 * Arabic has 6 categories
 
-Since CLDR has a strong set of pluralization rules defined for ~500 locales, each of which is supported by [ex_cldr for Elixir](https://hex.pm/ex_cldr), the ICU message format can reuse these pluralization rules in a simple and consisten fashion using the [plural format]{#Plural_Format}
+Since CLDR has a strong set of pluralization rules defined for ~500 locales, each of which is supported by [ex_cldr for Elixir](https://hex.pm/ex_cldr), the ICU message format can reuse these pluralization rules in a simple and consistent fashion using the [plural format]{#Plural_Format}.
```
Comment (alco) [inline README.md:48]: The `[plural format]{#Plural_Format}` link does not lead anywhere.

**Comment by kipcole9** [comment]
Thank you very much indeed for the PR, really appreciate it.  Will merge and also fix the relative links (which came from the ICU Message Format original content). This lib is definitely a work in progress butI do want to get it to a stable 1.0 release soon so thanks again for the help.

**Comment by alco** [comment]
Happy to contribute!

**Pull request merged**